### PR TITLE
Check osin deployment availability after other checks

### PR DESCRIPTION
**Pull request by enj**
See comment in handleVersion.

Signed-off-by: Monis Khan <mkhan@redhat.com>

@openshift/sig-auth 

/assign @stlaz @sallyom

### CODE REVIEW

File: pkg/operator2/status.go [hunk 1]
```diff
@@ -48,15 +48,10 @@ func setProgressingFalse(operatorConfig *operatorv1.Authentication) {
 }
 
 func setProgressingTrueAndAvailableFalse(operatorConfig *operatorv1.Authentication, reason, message string) {
+	setProgressingTrue(operatorConfig, reason, message)
+
```
Comment (sallyom) [inline pkg/operator2/status.go]: that was not there?? :| wth,Sally!

**Comment by wking** [comment]
Green :)  (although I don't understand this well enough to be able to review myself ;)

**Pull request merged**